packages/react-cosmos-normalize-props-proxy/src/__tests__/index.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import createNormalizePropsProxy from '../index';
+
+const NextProxy = () => {};
+const nextProxyNext = {};
+const nextProxy = {
+  value: NextProxy,
+  next: () => nextProxyNext,
+};
+const onComponentRef = jest.fn();
+const onFixtureUpdate = jest.fn();
+
+let NormalizePropsProxy;
+let componentRef;
+let childWrapper;
+let childProps;
+
+const renderProxy = fixture => {
+  componentRef = {};
+
+  jest.resetAllMocks();
+  NormalizePropsProxy = createNormalizePropsProxy();
+  const wrapper = shallow(
+    <NormalizePropsProxy
+      nextProxy={nextProxy}
+      component={() => {}}
+      fixture={fixture}
+      onComponentRef={onComponentRef}
+      onFixtureUpdate={onFixtureUpdate}
+    />,
+  );
+  childWrapper = wrapper.at(0);
+  childProps = childWrapper.props();
+
+  // Simulate rendering
+  childProps.onComponentRef(componentRef);
+};
+
+const commonTests = () => {
+  test('renders next proxy in line', () => {
+    expect(childWrapper.type()).toBe(NextProxy);
+  });
+
+  test('sends nextProxy.next() to next proxy', () => {
+    expect(childProps.nextProxy).toBe(nextProxyNext);
+  });
+
+  test('sends designated root level fixture fields to next proxy unmodified', () => {
+    expect(childProps.fixture.context).toEqual({});
+    expect(childProps.fixture.children).toEqual([]);
+    expect(childProps.fixture.state).toEqual({});
+    expect(childProps.fixture.reduxState).toEqual({});
```

Comment:
These checks can become false positives in time. How about creating some constants like `const mockContext = {}`, use them when constructing the fixture in `renderProxy` calls and then do strict checks like `expect(childProps.fixture.context).toBe(mockContext);`